data/blog/2024/2024-06-24-archiving-api-tools.md

```diff
@@ -0,0 +1,41 @@
+---
+id: 2024-06-24-archiving-api-tools
+author: julian
+title: 'API Tools (formerly Apigility) retirement'
+draft: false
+public: true
+created: '2024-06-24T11:00:00-01:00'
+updated: '2024-06-24T11:00:00-01:00'
+tags:
+  - meeting summary
+---
+
+On Monday, **6 May 2024**, the Technical Steering Committee for the Laminas Project held its monthly meeting.
+The topics discussed included the **API Tools (formerly Apigility)** alternatives.
+
+<!--- EXTENDED -->
+
+### Api Tools discussion
+
+The **Laminas Technical Steering Committee** decided in an initial meeting on **January 2023**, to mark
+laminas-api-tools as security only. The main reason is that laminas-api-tools main architecture is MVC and this
+architectural pattern is almost dead in favor of Middleware pattern (Mezzio). A new version of laminas-api-tools would
+be a new tool, without a migration path from current to the new one.
+
+All [laminas-api-tools packages](https://github.com/orgs/laminas-api-tools/repositories) will be marked as `abandoned`.
```

Comment:
> …will be marked as `abandoned`.

Not "abandoned", marked as security-only.

Reply:
There are currently no plans to follow these statutes. ([Here you will find the defined modes for maintenance.](https://github.com/laminas/technical-steering-committee/pull/175#issuecomment-2165101815))

Reply:
Those statuses are used to generate badges. But because require a new file (OSSMETADATA) and a new commit, it may be more complicated. 
Ok, i will use `security-only` in the article.

Reply:
> Those statuses are used to generate badges.

[I would follow @visto9259 here.](https://github.com/laminas/technical-steering-committee/pull/175#issuecomment-2165638407)

> Ok, i will use `security-only` in the article.

👍🏻